pages/common/meson.md

```diff
@@ -0,0 +1,24 @@
+# meson
+
+> SCons-like build system that use python as a front-end language and Ninja as a building backend.
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
> SCons-like build system that uses python as a front-end language and Ninja as a building backend.
```